src/include/OpenImageIO/attrdelegate.h

```diff
@@ -185,7 +186,14 @@ template<class C> class AttrDelegate {
         if (t == TypeString) {  // Attrib is a string? Return it.
             s = get<std::string>();
         } else if (t != TypeUnknown) {  // Non-string attrib? Convert.
-            char* buffer = new char[t.size()];
+            const int localsize = 64;   // Small types copy to stack, avoid new
+            char localbuffer[localsize];
+            char* buffer = localbuffer;
+            std::unique_ptr<char> allocbuffer;
```

Comment:
I think you want `std::unique_ptr<char[]>` (otherwise you mix `new char[]` with a plain `delete`)

Reply:
Quite right, I will fix.

Reply:
Quite right, I will fix.